Formula/sqldiff.rb

```diff
@@ -31,7 +31,7 @@ def install
     dbpath = testpath/"test.sqlite"
     sqlpath = testpath/"test.sql"
     sqlpath.write "create table test (name text);"
-    system "/usr/bin/sqlite3 #{dbpath} < #{sqlpath}"
+    system "sqlite3 #{dbpath} < #{sqlpath}"
```

Comment:
```suggestion
    sqlite3 = DevelopmentTools.locate("sqlite3")
    system "#{sqlite3} #{dbpath} < #{sqlpath}"
```

Reply:
We don't use this in any formula. I think we should have an audit that enforces the usage of `DevelopmentTools.locate` if wanted.

Reply:
It's only necessary if nothing is found in the path. And, to be honest, I can't think of a scenario where that would be the case.

Reply:
So my PR is fine then? :D